tests/test_utils/tests.py

```diff
@@ -914,3 +914,21 @@ def test_override_staticfiles_dirs(self):
         with self.settings(STATICFILES_DIRS=[test_path]):
             finder = get_finder('django.contrib.staticfiles.finders.FileSystemFinder')
             self.assertIn(expected_location, finder.locations)
+
+
+class DisallowedDatabaseQueriesTests(SimpleTestCase):
+    def runTest(self):
```

Comment:
I think I chose this pattern because I wanted to explicitly state the classes shouldn't reused for other tests but since this pattern is not used anywhere else in the suite I'll stick to the commonly used one.